packages/core/test/stringifyPolicy.spec.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-undef, @typescript-eslint/no-unused-vars, no-unused-expressions, no-extend-native */
```

Comment:
is there a fundamental change to this spec file from its former version at all, or was it just renamed basically?